meetups/2023/meetup-02/meetup-02.md

```diff
@@ -0,0 +1,44 @@
+# Rust Aarhus meetup #2 at [BRØLSTÆRK]
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+WE ARE STILL LOOKING FOR SPEAKERS. PLEASE CONTACT TUE HENRIKSEN FOR MORE INFORMATION.
+
+We are still working on the program, but it will be something like the following:
+
+- 18:00 - 18:10: Welcome by Tue Henriksen
+- 18:15 - 18:35: "When lifetimes give you lemons - an introduction to Rust lifetimes" and Q&A by Philip Kristoffersen (15+5 min)
+- 18:40 - 19:00: Lightning talks
+- 19:00 - 19:45: Light dinner sponsored by [BRØLSTÆRK]
+- 19:45 - 20:15: "Sudoku solving with Rust iterators for intellectual amusement" and Q&A by Niels Kristian Skovmand (25+5 min)
+- 20:20 - 20:40: "Ansible in Rust" and Q&A by Mikkel (15+5 min)
+- 20:40 - 21:00: Free play - bring your laptop!
+- ~21:00: Thank you for a hopefully wonderful and rewarding evening. Feedback for the meetup will be appreciated.
+
+## When lifetimes give you lemons - an introduction to Rust lifetimes
+[Philip Kristoffersen][pk] is a senior software consultant (Full-stack & Architect) at [BRØLSTÆRK] and works with everything from 3D Visualization to Microservices.
+
+In this talk we will first get everyone up to speed on what lifetimes are and why they are needed in Rust.
+Then we will look at how to use lifetimes, why they are cool and how to avoid common pitfalls.
+
+## Sudoku solving with Rust iterators for intellectual amusement
+[INFORMATION ABOUT THE SPEAKER]
+
+[ADD DESCRIPTION OF THE TALK HERE]
```

Comment:
```suggestion
## Solving Sudokus with Rust Iterators
Niels Kristian Skovmand is a Software Engineer at CrowdStrike, working with frontend web development on Falcon LogScale. He has been a Rust enthusiast ever since doing the Rustlings course 2 years ago 🦞

In the talk we will cover how to implement a sudoku solving iterator in Rust. We will cover the backtracking technique, how to build custom iterators, the benefits of laziness and the benefits for the consumer of the iterator.
```